sources/react/index-header.jsx

```diff
@@ -9,8 +9,7 @@ OR if you're looking to change now SVGs get output, you'll need to edit strings
  * WordPress dependencies
  */
 import { Component } from '@wordpress/element';
-import SVG from './svg';
-import Path from './path';
+import { SVG, Path } from '../../primitives';
```

Comment:
What is this pointing to? Where is `../../primitives` ?

Reply:
https://github.com/WordPress/gutenberg/tree/master/packages/components/src/primitives

We are trying to find the best way to start using all the primitives (HTML tags) on the web and their alternatives with React Native.

Reply:
This is the `WordPress/dashicons` project though. `WordPress/gutenberg` is a separate repository.

Reply:
Right, we need to find some workaround anyways. Should we put those primitives in their own package instead? It would be a similar case to `Component` import. I would appreciate your help with that as this is something we are still exploring. At the moment we made it work in React Native using [react-native-svg](https://github.com/react-native-community/react-native-svg).